jobs/config.json

```diff
@@ -8775,8 +8775,6 @@
       "--gcp-zone=us-central1-f",
       "--kubeadm=ci",
       "--kubernetes-anywhere-dump-cluster-logs=true",
-      "--kubernetes-anywhere-kubelet-ci-version=latest",
-      "--kubernetes-anywhere-kubernetes-version=ci/latest",
```

Comment:
You need a kubernetes version specified. Even the pull job has it specified.

Reply:
Thanks for catching that! I'll add that back in.